integration-tests/dataproc_test_case.py

```diff
@@ -0,0 +1,143 @@
+import json
+import logging
+import random
+import unittest
+import subprocess
+from threading import Timer
+
+import os
+
+DEFAULT_TIMEOUT = 10  # minutes
+
+
+class DataprocTestCase(unittest.TestCase):
+    DEFAULT_ARGS = {
+        "SINGLE": [
+            "--single-node",
+            "--worker-machine-type n1-standard-4",
```

Comment:
I don't think master/worker-machine-type is necessary. We should use the default (which is n1-standard-4, but not pin to it).